charts/sqnc-node/values.yaml

```diff
@@ -1,173 +1,388 @@
+    ##
+    isParachain: false
+    ## @param node.collator.relayChain [default: nil] Custom relay-chain name
+    ## @param node.collator.relayChainCustomChainspecUrl [default: nil] Download and use a custom relay-chain chainspec file from a URL
+    ## @param node.collator.relayChainDataSnapshotUrl [default: nil] Download and load relay-chain data from a snapshot archive HTTP URL
+    ## @param node.collator.relayChainDataSnapshotFormat [default: nil] The relay-chain snapshot archive format (tar or 7z)
+    ## @param node.collator.relayChainPath [default: nil] Path at which the chain database files are located (/data/polkadot/chains/${RELAY_CHAIN_PATH})
+    ## @param node.collator.relayChainDataKubernetesVolumeSnapshot [default: nil] Initialize the relay-chain data volume from a Kubernetes VolumeSnapshot
+    ## @param node.collator.relayChainDataGcsBucketUrl [default: nil] Sync relay-chain data files from a GCS bucket (eg. gs://bucket-name/folder-name)
+    ## @param node.collator.relayChainFlags [default: nil] Relay-chain node flags other than --name (set from the helm release name), --base-path and --chain
+    ##
+    ## relayChain: polkadot
+    ## relayChainCustomChainspecUrl: ""
+    ## relayChainDataSnapshotUrl: "https://dot-rocksdb.polkashots.io/snapshot"
+    ## relayChainDataSnapshotFormat: 7z
+    ## relayChainPath: ""
+    ## relayChainDataKubernetesVolumeSnapshot: ""
+    ## relayChainDataGcsBucketUrl: ""
+    ## relayChainFlags:
+    ##
+  ## @param node.podManagementPolicy [default: {}] The pod management policy to apply to the StatefulSet
+  ##
+  ## Set it to Parallel to launch or terminate all Pods in parallel, and not to wait for pods to become Running and Ready or completely terminated prior to launching or terminating another pod
+  ## podManagementPolicy: Parallel
+  ##
+  ## @param node.resources Resource requirements and limits for nodes
+  ##
+  ## Example
+  ## resources:
+  ##   requests:
+  ##     cpu: 2
+  ##     memory: 512Mi
+  ##   limits:
+  ##     cpu: 3
+  ##     memory: 1024Mi
+  ##
+  resources: {}
+  serviceMonitor:
+    ## @param node.serviceMonitor.enabled [default: false] If true, creates a Prometheus Operator ServiceMonitor
+    ##
+    enabled: false
+    ## @param node.serviceMonitor.namespace [default: nil] Prometheus namespace
+    ## @param node.serviceMonitor.internal [default: nil] Prometheus scrape interval
+    ## @param node.serviceMonitor.scrapeTimeout [default: nil] Prometheus scrape timeout
+    ##
+    ## Example
+    ## namespace: monitoring
+    ## interval: 10s
+    ## scrapeTimeout: 10s
+    ##
+  tracing:
+    ## @param node.tracing.enabled [default: false] If true, creates a Jaeger agent sidecar
+    ##
+    enabled: false
+  substrateApiSidecar:
+    ## @param node.substrateApiSidecar.enabled [default: false] If true, creates a Substrate API sidecar
+    ##
+    enabled: false
```

Comment:
@dblane-digicatapult I wonder if the node does support tracing with Jaeger secretly 🤔